### PR TITLE
Observability doc/guide updates for RHOCP 4.10

### PR DESCRIPTION
**Pull request by ellen-lau**
This PR contains updates to the Observability docs/guides for RHOCP 4.10.

### CODE REVIEW

File: doc/observability-deployment.adoc [hunk 1]
```diff
@@ -188,6 +187,38 @@ spec:
...
+    failureThreshold: 12
+    httpGet:
+      path: /health/started
+      port: 9443
+      scheme: HTTPS
+    initialDelaySeconds: 30
+    periodSeconds: 2
+    timeoutSeconds: 10
+...
+----
```
Comment (donbourne) [inline doc/observability-deployment.adoc:200]: I see you have the startupProbe section for the yaml below in the mpHealth-3.1+ section -- did you mean for it to be here as well?

File: doc/openshift-monitoring.adoc [hunk 2]
```diff
@@ -41,11 +41,11 @@ spec:
 ----
 +
 
-. Under Monitoring > Metrics, confirm that your service is being monitored by inserting your own queries specific to your app's metrics.
+. Under Observe > Metrics, confirm that your service is being monitored by inserting your own queries specific to your app's metrics.
```
Comment (donbourne) [inline doc/openshift-monitoring.adoc:44]: maybe say Observe > Metrics (formerly Monitoring > Metrics) ?

**Review by donbourne — APPROVED** [review]
Just a few minor comments.  Please have a look and update if you think they make sense.

File: doc/observability-deployment.adoc (hunk 1 above)
Comment (ellen-lau) [reply doc/observability-deployment.adoc:200]: nope, I'll remove that

**Pull request merged**